Makefile

```diff
@@ -48,7 +48,7 @@ unit: format lint
 
 acceptance: format lint
 	@echo "> Running acceptance tests..."
-	$(GOCMD) test -v -count=1 -parallel=1 -timeout=0 -tags=acceptance ./acceptance
+	COMPILE_PACK_WITH_VERSION=$(or ${COMPILE_PACK_WITH_VERSION}, 0.0.0) $(GOCMD) test -v -count=1 -parallel=1 -timeout=0 -tags=acceptance ./acceptance
```

Comment:
What's the purpose of setting a version on a short lived pack binary? It doesn't seem to be used for anything. Is it primarily aesthetic?

Reply:
Saw your later comment. Yes, it's needed for version-specific fixtures to be found.

Reply:
Also because the run combination for `make acceptance` is current-current-current the version-specific fixtures are not relevant, hence `0.0.0` is an okay value - but for `make acceptance-all` it seemed prudent to require either `COMPILE_PACK_WITH_VERSION` or `PACK_PATH` to be set to avoid confusing test failures when `pack version` is incorrect.